### ISSUE TITLE
Azure not properly deploying on the production branch

### ISSUE THREAD

**Issue by Ryu-From-Streets** [Ryu-From-Streets/classcompass#7]
After connecting the github repository to azure for deployment, deployment failed.

The issue seems to be because the workflow in ".github/workflows/production_course-compass.yml" is failing.
The failure is occurring during the npm install, build, and test.

I believe this is happening because by default, it expects the backend files to be in the parent directory, but we have backend and frontend separated in different directories, so the path it is executing the commands in is not correct, causing them to fail.
I plan to edit the workflow file to use the correct paths so that deployment succeeds. 

I don't expect this to take too long, but since I am using azure for the first time and am not too familiar with deployment, it may take longer than expected.

**HIGH PRIORITY**
**FIX ASAP**

**Comment by Ryu-From-Streets** [comment]
Azure takes a really long time to process all the files so fixing the azure hosting issue is taking a lot longer than expected :(

Currently, the paths work, and azure is saying it has deployed everything, but going to the url gives an application error.

Searching online, it seems the issue may be that the backend server is not listening to the port azure expects it to, so I've pushed some changes to the production build that let it use port 8080 which is the node.js default, so hopefully this will resolve the issue, or at the very least not give an application layer. 

My goal right now is just to be able to see the app running on the azure website. From there, we can figure out if there are integration errors or what not.

**Comment by Ryu-From-Streets** [comment]
Azure now properly deploys and hosts code on the production branch!

I added the start command "npm start --prefix ./backend" as based on what I saw online, azure likely was not displaying anything since it hasn't actually started running the application. I also changed the port to 1337 as that seemed to be the more common port used for mern stack web-apps hosted on azure.

It takes 30 minutes to redeploy code after a commit has been made on the production branch since it has to zip, unzip, and reprocess the almost 50,000 files that comprise our code, so since the deployment and hosting now works, I don't want to bother experimenting with port numbers anymore, since it works as it is.
